### ISSUE TITLE
Item display feature not working properly with "Discord Integration"

### ISSUE THREAD

**Issue by CPTProgrammer** [CPTProgrammer/ChatPlus#3]
Hi, thank you for working on fixing this issue.

I noticed you had updated the mod so I added it to my server to test but there is still one issue unfortunately.

I did some testing of my own and ChatPlus works fine when the default template config for Discord Integration is used.

As soon as you add your bot token and channel ID then it breaks the linking feature.

You can type [item] and it sends it to chat but it's just the text with no hover over info, I've provided a screenshot below.

Sending messages has been fixed though which is great!

Screenshot: https://egirl.rip/nbcB7E3ivu.png?key=kEIQHH1vfw6797

_Originally posted by @mrGRINCH in https://github.com/CPTProgrammer/ChatPlus/issues/1#issuecomment-1738705347_

**Comment by CPTProgrammer** [comment]
I found the reason. Discord-Integration redirected the `broadcast` function called in `handleDecoratedMessage` and changed the content in `signedMessage` to a `String` without `HoverableText`. 

I tried to fix this issue using Java's reflection mechanism, which worked perfectly, but would result in an incomplete fix for Issue #1, which **may** not trigger the forwarding function of message forwarding mods similar to it except Discord-Integration.

This change will be applied in v0.20.2. Thanks for your support!
